### ISSUE TITLE
Changing the speed unit has no effect

### ISSUE THREAD

**Issue by claustres** [danwild/leaflet-velocity#18]
Value is always in `m/s` and `m/s` is always displayed in the UI even if the `speedUnit: 'kt'` option is set.

This is linked to the fact that the merge of PR https://github.com/danwild/leaflet-velocity/pull/8 in commit https://github.com/danwild/leaflet-velocity/commit/38fe1a4f1ef1e03b605aadcf42ed3b59a4a0205e was not completly right, see https://github.com/danwild/leaflet-velocity/commit/38fe1a4f1ef1e03b605aadcf42ed3b59a4a0205e#commitcomment-25654602.

The correct code as seen [in my fork](https://github.com/weacast/leaflet-velocity/blob/master/src/js/L.Control.Velocity.js#L72) is the following:
```
if(gridValue && !isNaN(gridValue[0]) && !isNaN(gridValue[1]) && gridValue[2]) {
  htmlOut = "<strong>"+ this.options.velocityType +" Direction: </strong>"+  self.vectorToDegrees(gridValue[0],gridValue[1],this.options.angleConvention).toFixed(2) + "°" +
            ", <strong>"+ this.options.velocityType +" Speed: </strong>" + self.vectorToSpeed(gridValue[0],gridValue[1],this.options.speedUnit).toFixed(2) + this.options.speedUnit;
}
```

Since on my side the merge has been correct on my master it is not easy to create a specific PR for this, would you mind reporting it directly in your master branch ?

Thanks.

**Comment by SFSailor** [comment]
Good fix.  I did an "include" to make this work with my app.

**Comment by danwild** [comment]
Thanks, fixed in [cd565bd](https://github.com/danwild/leaflet-velocity/commit/519341188349f445252f4c6147b5e4944816ad99).

**Comment by SFSailor** [comment]
Thank you.  It's a great program.
